.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 14.x, 15.x]
```

Comment:
I don't think we need to test on all versions of Node. `Node 8.x` should be enough shouldn't it since we have always built and released this package using Node 8?

Reply:
We can narrow down sure, but building with newer versions to show we are compatible is nice to contributors as well.